packages/example_hardhat/README.md

```diff
@@ -1,79 +1,67 @@
 # <PACKAGE>
 
-*Include a high level description of this package here*
+*Include a high level description of this package here for this example*
+
+This project is based on the template generated by hardhat when run in an empty directory.
+
+This project demonstrates a basic Hardhat use case. It comes with a sample contract, a test for that contract, and a
+script that deploys that contract.
```

Comment:
Is this correct that it comes with a script that deploys the contract? I think we're taking this out right?